the-example-app/README.md

```diff
@@ -0,0 +1,22 @@
+What is `The Example App model`?
+================================
+
+The example app content model containts the model for all the example apps, demonstrating in a broader fashion some examples on how to use
+Contentful. If you want to see the model in action, we do have a hosted version here: https://contentful-example-app.herokuapp.com/
```

Comment:
remove filler "do":  If you want to see the model in action, we have a hosted version here: https://contentful-example-app.herokuapp.com/